_layouts/index.html

```diff
@@ -0,0 +1,38 @@
+{% include header.html %}
+
+{% include main_content.html %}
+
+{% include earlier_signatures.html %}
+
+<ul class="signatories" id="automated-signatories">
+  {% assign sorted_signatures =  site.signatures | sort: 'name' %}
+  {% for signature in sorted_signatures %}
+    <li>
+      {% if signature.url %}
+        <a href="{{ signature.link }}">{{ signature.name }}</a>
+      {% else %}
+        {{ signature.name }}
+      {% endif %}
+
+      {% if signature.affiliation %}
+      , {{ signature.affiliation }}
+      {% endif %}
+    </li>
```

Comment:
How do we get the github username? Should that be an optional front matter field in the template?